helix-term/src/commands/typed.rs

```diff
@@ -656,7 +656,7 @@ pub fn write_all_impl(
     force: bool,
     write_scratch: bool,
 ) -> anyhow::Result<()> {
```

Comment:
The changes to write_all_impl should be reset: changing `&'static str` to `String` will cause a needless allocation and we shouldn't be discarding errors when the `force` flag is passed.

Reply:
> we shouldn't be discarding errors when the `force` flag is passed.

It was like this before: https://github.com/helix-editor/helix/blob/c7e9e94f007e04836d134cd3e53f71c38b7ba1ec/helix-term/src/commands/typed.rs#L723

Should I just revert it?

Reply:
im just going to revert it since in the end no logic was changed in this function